### ISSUE TITLE
Global hotkeys move the virtual cursor with VoiceOver

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/forces-and-motion-basics#443]
Over slack, @arouinfar reported that global hotkeys also move the virtaul cursor. Testing was done with macOS VoiceOver.

I tested with NVDA and JAWS and could not reproduce the issue.

I wonder if an event.preventDefault will fix the problem. This issue is to track testing. If we find something that works, lets open an issue in scenery to fix generally.

**Comment by arouinfar** [comment]
Thanks for looking into it @jessegreenberg. Unfortunately, the cursor is still returning to the top of the PDOM after using the go/pause global hotkeys. Steps to reproduce:

1. To save some time, set up the sim before turning on VoiceOver. Add a puller or two to the rope and check 'Speed'.
2. Turn on VO, and use VO keys to navigate to 'Speed' heading.
3. Use hotkey for 'Go!' (Opt+G) to start tug-of-war
4. Press VO+right arrow to read the next line, which should be about the speed.
5. VO will read the Scene Summary, "The Net Force Screen changes as you play with it..."

**Comment by jessegreenberg** [comment]
Darn. Ok thanks @arouinfar. Ill revert this change and will need to investigate a bit more carefully.

**Comment by jessegreenberg** [comment]
Thanks @arouinfar, sounds good. @KatieWoe was not on the latest version either when we tested, though I am not sure what version it was.

**Comment by arouinfar** [comment]
Unfortunately, upgrading to the latest macOS did not fix this issue.

**Comment by arouinfar** [comment]
> Does this happen if you use the Go button normally with the keybaord? 

No, if I tab to the Go button to use it, focus stays on the GoPauseButton, as does the virtual cursor.

> Does this happen in any other sim that does NOT have these global hotkeys? 

Membrane Transport and Greenhouse Effect both have two global hotkeys (PlayPauseButton and ResetAllButton). When using these hotkeys, my cursor stays put. Before I updated macOS, the ResetAllButton hotkey would move my cursor back to the top of the page. I also tried using Opt+G in these sims to see if it was somehow interfering with VoiceOver controls, but the hotkey does nothing.

**Comment by jessegreenberg** [comment]
This issue may depend on *where* the virtual cursor is when you press the Opt + G key. If the virtual cursor is on the "Speed" heading, that may fully re-render when the accessible paragraph below it changes.

**Comment by KatieWoe** [comment]
On main, and I see exactly what is described in https://github.com/phetsims/forces-and-motion-basics/issues/443#issuecomment-3294348954 when following those steps on main

**Comment by jessegreenberg** [comment]
Ah, great @KatieWoe. OK thanks. Lets schedule a time to investigate.

**Comment by jessegreenberg** [comment]
I tried looking at `PDOMUtils.insertElements` to understand re-renders. With `accessibleParagraph`, I do see that the content for the speedometer gets re-rendered every change. When it is changed to a static string, I see that that element stops getting re-rendered.

This is all good news, but means that we have not identified the cause of this problem because the problem persists even with static strings.

I just did a final test with NVDA and cannot reproduce on that device.

Our tests in https://github.com/phetsims/forces-and-motion-basics/issues/443#issuecomment-3321255626 indicate that this may still be a scenery problem, but I am not sure what else to look for. And I wonder if continued exploration is a good use of time.

I would recommend that we note this behavior in the known screen reader bugs and close with wontfix. @arouinfar does that sound reasonable or do you think this is something we should continue to investigate?

**Comment by jessegreenberg** [comment]
OK, thanks! Closing.
